Loop/Managers/LoopDataManager.swift

```diff
@@ -488,6 +488,26 @@ extension LoopDataManager {
             }
         }
     }
+    
+    /// Take actions to address how insulin is delivered when the CGM data is unreliable
+    ///
+    /// An active high temp basal (greater than the basal schedule) is cancelled when the CGM data is unreliable.
+    func recievedUnreliableCGMReading() {
+        guard case .tempBasal(let tempBasal) = basalDeliveryState,
+              let scheduledBasalRate = basalRateSchedule?.value(at: now()),
+              tempBasal.unitsPerHour > scheduledBasalRate else
+        {
+            return
+        }
+              
+        // Setting a temp basal with 0 duration effectively cancels that temp basal
+        let basalRecommendation = TempBasalRecommendation(unitsPerHour: scheduledBasalRate, duration: .minutes(0))
```

Comment:
[nit] So, using `scheduledBasalRate` is irrelevant here since it is 0 duration? This might be confusing.

Also, `.minutes(0)` can just be `0`.

Reply:
Since the intent is to cancel any temp basal, I thought it would be clearer to use the scheduled basal then use some arbitrary value. 

I'll update to `0`?

Reply:
Actually, I think the better solution is to use `TempBasalRecommendation.cancel`. Check out the `TempBasalRecommendation.swift` file.